EssentialFeed/EssentialFeedTests/Image Comments/API/ImageCommentsMapperTests.swift

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+
+class ImageCommentsMapperTests: XCTestCase {
+	func test_map_throwsErrorOnNon200HTTPResponse() throws {
```

Comment:
It should be `Non2xx` instead of `Non200`
```suggestion
	func test_map_throwsErrorOnNon2xxHTTPResponse() throws {
```

Reply:
done